statsmodels/genmod/generalized_estimating_equations.py

```diff
@@ -728,6 +733,137 @@ def cluster_list(self, array):
             return [np.array(array[self.group_indices[k], :])
                     for k in self.group_labels]
 
+    def score_test(self, submodel, tol=1e-6):
```

Comment:
maybe rename to `compare_score_test`

I haven't looked at any details yet (still recovering from moving to a new house), but we should get consistent signature and interpretation for score_test.

I'm not sure whether `score_test` should be attached to the restricted model, the unrestricted model or both, depending on the options. (eg. I have an option for variable addition when the restricted model is available.)

for LR test we use `compare_lr_test` that takes a nested model.